diff -urN cfengine-3.5.3.orig/libutils/platform.h cfengine-3.5.3/libutils/platform.h
--- libutils/platform.h	2013-12-09 13:13:14.000000000 +0100
+++ libutils/platform.h	2014-05-31 00:06:11.000000000 +0200
@@ -569,6 +569,7 @@
 #if !HAVE_DECL_OPENAT
 int openat(int dirfd, const char *pathname, int flags, ...);
 int fstatat(int dirfd, const char *pathname, struct stat *buf, int flags);
+int fchownat(int dirfd, const char *pathname, uid_t owner, gid_t group, int flags);
 #ifndef AT_SYMLINK_NOFOLLOW
 #define AT_SYMLINK_NOFOLLOW 0x1000
 #endif
